tests/components/nexia/util.py

```diff
@@ -17,10 +17,11 @@ async def async_init_integration(
     hass: HomeAssistant,
     skip_setup: bool = False,
     exception: Exception | None = None,
+    *,
+    house_fixture="nexia/mobile_houses_123456.json",
 ) -> MockConfigEntry:
     """Set up the nexia integration in Home Assistant."""
 
-    house_fixture = "nexia/mobile_houses_123456.json"
```

Comment:
At some point, in a future PR, we should refactor this to not use fixtures and mock the nexia library API.  This integration is a bit older though so its a bit more difficult to change how the tesitng works now.